mantle/kola/tests/coretest/core.go

```diff
@@ -378,3 +382,79 @@ func TestMachineID() error {
 	}
 	return nil
 }
+
+// TestRHCOSGrowpart tests whether rhcos-growpart.service was run successfully
+// and check that filesystem size has been grown to at least 15 GB.
+func TestRHCOSGrowpart() error {
+	// check that rhcos-growpart.service was run and exited normally
+	err := checkGrowpart("rhcos-growpart.service")
+	if err != nil {
+		return err
+	}
+
+	// check that filesystem size is >= 15 GB
+	// 16106127360 Bytes = 15 Gigabytes
+	err = checkFilesystemSize(16106127360)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// TestFCOSGrowpart tests whether ignition-ostree-growfs.service was run successfully
+// and check that filesystem size has been grown to at least 7 GB.
+func TestFCOSGrowpart() error {
+	// check that ignition-ostree-growfs.service was run and exited normally
+	err := checkGrowpart("ignition-ostree-growfs.service")
+	if err != nil {
+		return err
+	}
+
+	// check that filesystem size is >= 7 GB
+	// 7516192768 Bytes = 7 Gigabytes
+	err = checkFilesystemSize(7516192768)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func checkGrowpart(unit string) error {
+	// value of MESSAGE_ID= is the SD_MESSAGE_UNIT_STARTED macro's value from
+	// https://github.com/systemd/systemd/blob/master/src/systemd/sd-messages.h
```

Comment:
Maybe a comment here mentioning that for oneshot services, `STARTED` means the script ran and exited successfully?